### PR TITLE
Removed Other from TV Guide data providers

### PR DESCRIPTION
**Pull request by NathanPickard**
**Changes**

Remove 'Other' from TV Guide Data providers

**Issues**

### CODE REVIEW

File: src/controllers/livetvstatus.js [hunk 1]
```diff
@@ -168,9 +168,6 @@ define(["jQuery", "globalize", "scripts/taskbutton", "dom", "libraryMenu", "layo
         }), menuItems.push({
             name: "Xml TV",
             id: "xmltv"
-        }), menuItems.push({
-            name: globalize.translate("ButtonOther"),
-            id: "other"
```
Comment (dkanada) [inline src/controllers/livetvstatus.js]: The corresponding action for this key needs to be removed as well, along with any unused translations.

File: src/controllers/livetvstatus.js [hunk 2]
```diff
@@ -168,17 +168,12 @@ define(["jQuery", "globalize", "scripts/taskbutton", "dom", "libraryMenu", "layo
...
         }), require(["actionsheet"], function(actionsheet) {
             actionsheet.show({
                 items: menuItems,
                 positionTo: button,
                 callback: function(id) {
-                    "other" == id ? Dashboard.alert({
-                        message: globalize.translate("ForAdditionalLiveTvOptions")
```
Comment (dkanada) [inline src/controllers/livetvstatus.js]: One more minor change, the string `ForAdditionalLiveTvOptions` is probably also unused.

File: src/livetvtuner.html [hunk 3]
```diff
@@ -63,7 +63,6 @@ <h1 class="sectionTitle">${HeaderLiveTvTunerSetup}</h1>
                 </div>
 
                 <p class="drmMessage hide">${DrmChannelsNotImported}</p>
-                <p class="otherOptionsMessage hide" style="margin:2em 0;">${ForAdditionalLiveTvOptions}</p>
```
Comment (dkanada) [inline src/livetvtuner.html]: Apologies, I hadn't noticed this usage. The Live TV page may throw an error and crash if this element is missing so we need to see if that controller still looks for this and remove the check.

**Comment by dkanada** [comment]
@grafixeyehero can you make sure these changes make it into your pull request? It might cause merge issues, sorry.

**Comment by grafixeyehero** [comment]
> 
> 
> @grafixeyehero can you make sure these changes make it into your pull request? It might cause merge issues, sorry.

Done

**Pull request merged**